lib/completion-templates.ts

```diff
@@ -47,3 +47,23 @@ _{{app_name}}_yargs_completions()
 compdef _{{app_name}}_yargs_completions {{app_name}}
 ###-end-{{app_name}}-completions-###
 `;
+
+export const completionFishTemplate = `### {{app_name}} completion - begin. generated by omelette.js ###
+#
+# yargs command completion script
+#
+# Installation: {{app_name}} {{completion_command}} >> ~/.config/fish/config.fish.
```

Comment:
I noticed a completions folder when I installed fish to try this out. Would it be more appropriate to suggest writing the completion into `~/.config/fish/completions/{{app_name}}.fish`?

https://fishshell.com/docs/current/completions.html#where-to-put-completions

Reply:
I checked what omelette does, which uses the lowest